dsc/scriptResource.md

```diff
@@ -3,22 +3,11 @@ ms.date:  06/12/2017
 keywords:  dsc,powershell,configuration,setup
 title:  DSC Script Resource
 ---
-
 # DSC Script Resource
 
-
 > Applies To: Windows PowerShell 4.0, Windows PowerShell 5.0
 
-The **Script** resource in Windows PowerShell Desired State Configuration (DSC) provides a mechanism to run Windows PowerShell script blocks on target nodes. The `Script` resource has `GetScript`, `SetScript`, and `TestScript` properties. These properties should be set to script blocks that will run on each target node.
-
-The `GetScript` script block should return a hashtable representing the state of the current node. The hashtable must only contain one key `Result` and the value must be of type `String`. It is not required to return anything. DSC doesn't do anything with the output of this script block.
-
-The `TestScript` script block should determine if the current node needs to be modified. It should return `$true` if the node is up-to-date. It should return `$false` if the node's configuration is out-of-date and should be updated by the `SetScript` script block. The `TestScript` script block is called by DSC.
-
-The `SetScript` script block should modify the node. It is called by DSC if the `TestScript` block return `$false`.
-
-If you need to use variables from your configuration script in the `GetScript`, `TestScript`, or `SetScript` script blocks, use the `$using:` scope (see below for an example).
-
+The **Script** resource in Windows PowerShell Desired State Configuration (DSC) provides a mechanism to run Windows PowerShell script blocks on target nodes. The `Script` resource uses `GetScript`, `SetScript`, and `TestScript` properties that contain script blocks you define to perform the corresponding DSC state operations.
```

Comment:
The second Script should be bold, not in codefence since the standard is to bold the resource name itself.

Reply:
Addressed